docs/introduction.rst

```diff
@@ -116,4 +116,27 @@ Other benefits for developers include:
   dependencies, which can be used by applications, and which can take a lot
   of the work out of application development.
 
+
+In general Flatpak is best suited suited for desktop applications and
+while command line applications also work, it may not be suited in some
+cases:
+
+- Applications needs to elevate priviledges using ``su, sudo, pkexec``
+  etc. Flatpak cannot run in SUID binaries inside the sandbox.
+- Application needs to read ``/proc`` from host or have unfiltered access
+  to processes. This is not allowed as Flatpak has a private proc.
+- Application uses a syscall that is blocklisted by Flatpak's seccomp
+  filter. For example, Flatpak won't allow spawning sub-namespaces
+  in the sandbox.
+- Kernel modules, drivers, daemons are non application packages and
+  won't work inside a Flatpak.
```

Comment:
`daemon` is a vague term. You can ship a dbus service as a flatpak that has no UI.

Reply:
This got covered by the systemd para below. I'll remove it.